tests/tests/test_stdcm.py

```diff
@@ -266,7 +266,75 @@ def test_max_running_time(small_scenario: Scenario, fast_rolling_stock: int):
     r = requests.post(url, json=payload)
     response = r.json()
     assert r.status_code == 200
-    assert response == {"status": "path_not_found"}
+    assert response == {
+        "status": "conflicts",
+        "conflicts": [
+            {
+                "conflict_type": "Spacing",
+                "start_time": "2024-01-01T07:30:00Z",
+                "end_time": "2024-01-01T16:00:00Z",
+                "train_ids": [0],
```

Comment:
It's weird that there is a train_id there. The timetable is supposed to be empty with just work schedules. 

Is it the train id of the "fake" new train?

Reply:
`0` is the id of `fake_train`

Reply:
I don't think it should be included in the response then.

By the way, `0` may also be the id of a valid train, right? What happens if the new trains conflicts with a scheduled train with id=0?

Reply:
As discussed, I removed the ID of the fake train from the list.

Reply:
Nice, thanks.

What about the case where there's also a train with id=0 in the database?

Maybe I'll ping someone more familiar with editoast because I'm not 100% confident about how ids work (ping @woshilapin @leovalais). Train IDs are incremental integers starting at 0 included, right? Could we just replace the value here with a `-1` or would that break things? (it shouldn't be saved in db)

Reply:
We use `GENERATED BY DEFAULT AS IDENTITY` in Postgres, which starts generating IDs from 1, so we don’t have an issue with id=0 being generated by the database.
For more details: https://www.postgresql.org/docs/current/ddl-identity-columns.html